plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -511,10 +511,18 @@ def set_env
       end
 
       env_containers.each do |c|
+        extra = all
         env = (c[:env] ||= [])
-        env.concat all
 
-        # unique, but keep last elements
+        # keep container env var if requested, so static+plugin env can be overwritten
+        if keep = samson_container_config(c, :"samson/keep_env_var").to_s.split(",").presence
```

Comment:
can we split this on whitespace? Then we can use the nice `>` or `|` YAML operators?

Reply:
yeah, can do both